pkgs/tools/misc/fastfetch/default.nix

```diff
@@ -0,0 +1,128 @@
+{ fetchFromGitHub
```

Comment:
The order is
-  lib
- stdenv
- fetchers
- all the rest in ASCIIbetical order
- if there are dependencies on Darwin, they will be the last ones

Reply:
Where is the documentation for some of this stuff? I couldn't seem to find it from the CONTRIBUTING.md or https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-new-packages